src/kademlia/msg.cpp

```diff
@@ -36,11 +36,13 @@ POSSIBILITY OF SUCH DAMAGE.
 
 namespace libtorrent { namespace dht {
 
-namespace dht_detail {
-
-bool verify_message(bdecode_node const& message, key_desc_t const desc[]
-	, bdecode_node ret[], int size, char* error, int error_size)
+bool verify_message_impl(bdecode_node const& message, span<key_desc_t const> desc
```

Comment:
I was looking at making this exact change a while ago. I can't remember why I gave up.